packages/compartment-mapper/test/scaffold.js

```diff
@@ -291,13 +291,15 @@ export function scaffold(
       };
 
       const sourceMaps = new Set();
+      const sourceMapLog = [];
       const sourceMapHook = (sourceMap, { sha512 }) => {
         sourceMaps.add(sha512);
-        t.log(sha512, sourceMap);
+        sourceMapLog.push(['+', sha512, sourceMap]);
```

Comment:
Made the log silent until it's needed because with even more lines added below it was getting annoying.